vendor/src/github.com/seccomp/libseccomp-golang/seccomp.go

```diff
@@ -20,7 +20,7 @@ import (
 
 // C wrapping code
 
-// #cgo LDFLAGS: -lseccomp
+// #cgo pkg-config: libseccomp
```

Comment:
>  although the test did not fail on that with the gccgo build 

Perhaps because Dockerfile.gccgo has no "seccomp" in DOCKER_BUILDTAGS?

> Maybe best to remove the pkgconfig part?

To be prepared for enabling seccomp on s390x I would vote for this.